tests/unit/test_utils.py

```diff
@@ -27,6 +27,7 @@ def test_get_col_name_longer():
     assert actual == "local"
 
 
+@pytest.mark.skip(reason="Ignore")
```

Comment:
We can move away from mock to actual results here since the initial work was to introduce the concept, staying away from introducing changes at this point. Once the approach is approved we can make changes to the test to reflect the expected behaviour.

Reply:
i like the idea